modules/300-prometheus/templates/prometheus/longterm/prometheus.yaml

```
@@ -133,8 +133,18 @@ spec:
       checksum/kube-rbac-proxy: {{ include "helm_lib_kube_rbac_proxy_ca_certificate" (list . "") | sha256sum }}
   secrets:
   - prometheus-api-client-tls
-  {{- include "helm_lib_node_selector" (tuple . "monitoring") | nindent 2 }}
-  {{- include "helm_lib_tolerations" (tuple . "monitoring") | nindent 2 }}
+  {{- if .Values.prometheus.longtermNodeSelector }}
+  nodeSelector: {{ .Values.prometheus.longtermNodeSelector | toJson }}
+  {{- else }}
+      {{- include "helm_lib_node_selector" (tuple . "monitoring") | nindent 2}}
+  {{- end }}
+  {{- if .Values.prometheus.longtermTolerations }}
+  tolerations:
+      {{ .Values.prometheus.longtermTolerations | toYaml | nindent 2}}
+  {{- else }}
+      {{- include "helm_lib_tolerations" (tuple . "monitoring") | nindent 2 }}
```

```suggestion
    {{- include "helm_lib_node_selector" (tuple . "monitoring") | nindent 2}}
  {{- end }}
  {{- if .Values.prometheus.longtermTolerations }}
  tolerations:
    {{ .Values.prometheus.longtermTolerations | toYaml | nindent 2}}
  {{- else }}
    {{- include "helm_lib_tolerations" (tuple . "monitoring") | nindent 2 }}
```
Our standard tabulation is two spaces. Sorry, didn't focus on that.

---

Ok. Got it. Thanks.

Does it means, that format above isn't correct?
```yaml
  {{- if not ( .Values.global.enabledModules | has "vertical-pod-autoscaler-crd") }}
        {{- include "longterm_config_reloader_resources" . | nindent 8 }}
  {{- end }}
```